projects/Mallard/src/screens/lightbox.tsx

```diff
@@ -74,7 +74,7 @@ const styles = StyleSheet.create({
 
 const LightboxScreen = () => {
 	const navigation = useNavigation();
-	const route = useRoute<RouteProp<RootStackParamList, 'Lightbox'>>();
+	const route = useRoute<RouteProp<MainStackParamList, 'Lightbox'>>();
```

Comment:
Not introduced in this PR but should we use `RouteNames.Lightbox` here?

Reply:
Yeah good shout! Updated.